src/lightning/fabric/CHANGELOG.md

```diff
@@ -124,6 +124,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Added support for saving and loading stateful objects other than modules and optimizers ([#18513](https://github.com/Lightning-AI/lightning/pull/18513))
 
 
+- Added support for passing the process group to the `FSDPStrategy` ([#18583](https://github.com/Lightning-AI/lightning/pull/18583))
```

Comment:
Is an addition of this PR? Wasn't this already possible?

Reply:
I guess for Fabric's FSDP yes, this was already possible. For PyTorch FSDP I added it because previously, the process group was automatically set and not taken from kwargs.